examples/ui.apps/src/main/content/jcr_root/apps/forms-components-examples/components/form/verticaltabs/_cq_template.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0"
+    jcr:primaryType="nt:unstructured"
+    jcr:title="Vertical Tab"
```

Comment:
minor: Plural noun to be consistent everywhere. Vertical Tabs

Reply:
done